<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>87</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50, 50, 50);color: rgb(255, 255, 255);selection-background-color:rgb(140, 140, 140)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="MainFrame">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(40,40,40);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0,0,0,0">
          <item>
           <widget class="QLabel" name="name">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>My Recording</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="RecordingInfoLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout" stretch="1">
              <property name="spacing">
               <number>4</number>
              </property>
              <item>
               <widget class="QLabel" name="imageCount">
                <property name="font">
                 <font>
                  <family>Tahoma</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>-</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="spacing">
               <number>4</number>
              </property>
              <item>
               <widget class="QLabel" name="duration">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>70</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>70</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Tahoma</family>
                  <pointsize>10</pointsize>
                  <stylestrategy>PreferDefault</stylestrategy>
                 </font>
                </property>
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="fps">
                <property name="minimumSize">
                 <size>
                  <width>20</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>20</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Tahoma</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>0</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label">
                <property name="font">
                 <font>
                  <family>Tahoma</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>fps</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="audioCheck">
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::ActionsContextMenu</enum>
            </property>
            <property name="text">
             <string>Audio</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="timeDisplay">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>115</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>115</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Courier</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="play">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>images/playBtn.png</normaloff>images/playBtn.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="record">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>images/recordBtn.png</normaloff>images/recordBtn.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSlider" name="timeSlider">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="maximum">
           <number>240</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="timeSliderProgress">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>10</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">selection-background-color: rgb(140, 0, 0);</string>
          </property>
          <property name="maximum">
           <number>240</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="textVisible">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>timeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>timeSliderProgress</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeSlider</sender>
   <signal>rangeChanged(int,int)</signal>
   <receiver>timeSliderProgress</receiver>
   <slot>setRange(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
